src/features/Chat/Chat.module.scss

```
@@ -18,10 +18,15 @@
     border-bottom: 1px solid var(--border);
     display: inline-flex;
     @include m.flex-box(center, center);
-    padding: 20px 20px 16px 20px;
-    @include m.text-style('xl', 700);
+    padding: 10px 16px 8px 16px;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)